mailer.rst

```diff
@@ -1563,10 +1563,35 @@ encrypter that automatically applies to all outgoing messages:
         return static function (FrameworkConfig $framework): void {
             $mailer = $framework->mailer();
             $mailer->smimeEncrypter()
-                    ->certificate('%kernel.project_dir%/var/certificates/smime.crt')
+                    ->repository('app.my_smime_encrypter')
             ;
         };
 
+The ``repository`` option must be a service ID that implements
+:class:`Symfony\\Component\\Mailer\\EventListener\\SmimeCertificateRepositoryInterface`::
+
```

Comment:
The purpose of the method `SmimeCertificateRepositoryInterface::findCertificatePathFor` is to resolve the certificate file path for a given email address, as the S/MIME encrypter expects a valid local file path to the certificate.